Makefile

```diff
@@ -2,14 +2,14 @@
 
 install:
 	pip install -r requirements.txt
-	python -m pip install --upgrade pip
+	python3 -m pip install --upgrade pip
```

Comment:
leave here `python` please instead of `python3`. It seems to me it was my change but we have `python` below (in the run command) so let's use the same call here